release notes/upcoming.md

```diff
@@ -116,3 +116,6 @@ Thanks to @AndriiChuzhynov for implementing this! (#766)
 * Setup data: Distinguish between `undefined` (when there is no `setup()` function or when it doesn't return anything) and `null` (when `setup()` explicitly returns `null`) values for the setup `data` that's passed to the default function and `teardown()` (#799)
 * Setup data: Prevent data races by having each VU have its own independent copy of the setup data (#799)
 * HAR converter: Support HAR files that don't have a `pages` array (#806)
+* UX: Instead of panicing on some operations in the init context we not return an error that the given
```

Comment:
typos
- `panicing` -> `panicking`
- `we not return` -> `we now return`
- `requests(batched or not)` -> `requests (batched or not)`
also (#811) should be at the end, see the other messages